Utilities/DataLoading/EdFi.LoadTools/BulkLoadClient/Application/BulkLoadClientConfiguration.cs

```diff
@@ -138,16 +138,16 @@ public static BulkLoadClientConfiguration Create(IConfiguration configuration)
                 : Path.GetFullPath(configuration.GetValue<string>("Folders:Working"));
 
             var xsdFolder = string.IsNullOrEmpty(configuration.GetValue<string>("Folders:Xsd"))
-                ? Path.Combine(workingFolder, "Schemas")
+                ? Path.Combine(workingFolder, "xsd")
                 : Path.GetFullPath(configuration.GetValue<string>("Folders:Xsd"));
 
             return new BulkLoadClientConfiguration
             {
                 ConnectionLimit = configuration.GetValue("Concurrency:ConnectionLimit", 100),
                 DataFolder = configuration.GetValue("Folders:Data", currentDirectory),
-                DoNotValidateXml = !configuration.GetValue<bool>("ValidateSchema"),
+                DoNotValidateXml = configuration.GetValue<bool>("ValidateSchema"),
```

Comment:
most of the changes to this file seem to be reverting recent changes. was that intentional?